cfme/tests/intelligence/reports/test_crud.py

```diff
@@ -147,8 +117,6 @@ def test_menuwidget_crud(appliance):
     w.delete()
 
 
-@pytest.mark.sauce
-@pytest.mark.tier(3)
 @pytest.mark.meta(blockers=[BZ(1656413), BZ(1667064)])
 def test_reportwidget_crud(appliance):
```

Comment:
[1656413](https://bugzilla.redhat.com/show_bug.cgi?id=1656413) is verified.
Please remove from blockers.